WcaOnRails/lib/results_validators/validation_error.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module ResultsValidators
+  ERRORS_AND_WARNING_KINDS = [:events, :persons, :results, :rounds, :scrambles].freeze
```

Comment:
Actually I intended to use them in the yet-to-come validation view, to be able to show the user the list of warnings/errors per kind (pretty much like it's done now on the check results page, but instead of hardcoding the list of kinds in the view it would just iterate over that variable).

I'll remove it for now but it will surely come again soon :)